Yona/Yona/AppDelegate.swift

```diff
@@ -69,7 +68,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate,URLSessionDelegate {
         return true
         
     }
-        
+    
+    func initializeCrashlytics(){
+        do{
+            let fabricResourceFile = R.file.fabricApikey()
+            let fabricAPIKey = try String(contentsOf: fabricResourceFile!)
+            let fabricAPIKeyTrimmed = fabricAPIKey.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
+            if !(fabricAPIKeyTrimmed.isEmpty) {
+                isFabricFilePresent = true
+                Crashlytics.start(withAPIKey: fabricAPIKeyTrimmed)
```

Comment:
Move this one before the previous line, so you only set the flag when it's successfully loaded. With that meaning, I propose to rename the flag into ``isCrashlyticsInitialized``.

Reply:
Done.